docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       '
 
   # Transaction validator
-  validator:
+  validator-1:
```

Comment:
changed the naming scheme to `<service>-<number>` to make it easier to show on demo that we're running in multinode network.